### DISCUSSION TITLE
Feature Request: Option to display alert summary for the alert list visualization on the dashboard

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#65441] [Dashboards, Panels & Visualizations]
Currently, the [Alert list visualization](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/alert-list/) only displays the labels by default. The summary and description is hidden but and can only be viewed if the alert has been expanded. This makes it difficult for me to quickly identify which alerts require attention and what the problem is.

My suggestion is to add an option to display the alert summary instead of the labels by default. This would make it much easier for me to see everything all at once.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82773. Feel free to continue the discussion around this there. Thank you!
